src/Table.php

```diff
@@ -364,10 +364,15 @@ public function renderView()
     /**
      * Render individual row. Override this method if you want to do more
      * decoration.
+     *
+     * @param array|\atk4\data\Model $m
      */
-    public function renderRow()
+    public function renderRow($m = null)
     {
-        $this->t_row->set($this->model);
+        $this->t_row = new Template($this->t_row_master);
```

Comment:
i don't like that... specifically tried to avoid creating any objects during table iteration. That will slow down your big tables  A LOT.

Reply:
hmm... you are right about creating object on each iteration, that should slow down.
will cloning on each iteration be faster?
like $t_row = clone $t_row_master ?